helpers/pathmap.py

```diff
@@ -13,6 +38,21 @@ def __init__(self, *args, **kwargs):
         # Original value indicator
         self._ORIG = "\\*__orig__*//"
 
+    def resolve_path(self, path: str, ancestors=None):
```

Comment:
Great move! Quickly looking up the type for `ancestors` and `resolve_path`:

```suggestion
    def resolve_path(self, path: str, ancestors:int | None=None) -> str | None:
```